hci/event/events/hci_commands_complete/hci_read_rssi.py

```diff
@@ -0,0 +1,26 @@
+from struct import unpack_from
+
+from .. import HCI_CommandComplete
+
+
+class HCI_ReadRssi(HCI_CommandComplete):
```

Comment:
Since `MTU` is written uppercase in other commands, I think `RSSI` should also be in uppercase letters.